src/main/kotlin/no/nav/syfo/service/DialogmoteInnkallingVarselService.kt

```diff
@@ -193,11 +202,12 @@ class DialogmoteInnkallingVarselService(val senderFacade: SenderFacade, val dial
     fun dataToDialogmoteInnkallingArbeidstakerData(data: Any?): DialogmoteInnkallingArbeidstakerData {
```

Comment:
Ikke nødvendig, men for å få litt mer struktur kan du legge inn `varselUuid: VarselUuid `som et felt her
https://github.com/navikt/esyfovarsel/blob/master/src/main/kotlin/no/nav/syfo/kafka/consumers/varselbus/domain/EsyfovarselHendelse.kt#L28
og f.eks lage en data class `VarselUuid` med et `uuid: String` felt.
Da kan det hentes ut på samme måte som her:
https://github.com/navikt/esyfovarsel/blob/master/src/main/kotlin/no/nav/syfo/service/OppfolgingsplanVarselService.kt#L95
slik at det blir litt likere over koden. E.g.
```
private fun EsyfovarselHendelse.dataToDialogmoteInnkallingArbeidstakerData(): DialogmoteInnkallingArbeidstakerData {
  return this.data?.let {
    try {
      ...
    } catch (e: IOException) { ... }
  } ?: throw MissingArgumentException("EsyfovarselHendelse mangler 'data'-felt")
}

```